api/v1/status_routes_test.go

```diff
@@ -80,16 +86,43 @@ func TestGetStatus(t *testing.T) {
 }
 
 func TestPatchStatus(t *testing.T) {
+	t.Parallel()
+
 	testdata := map[string]struct {
-		StatusCode int
-		Status     Status
+		StatusCode     int
+		ExpectedStatus Status
+		Payload        []byte
```

Comment:
```suggestion
		ExpectedStatusCode     int
		ExpectedStatus Status
		Payload        []byte
```
I would prefer to change it in ExpectedStatusCode to be consistent with `ExpectedStatus`